src/pudl/transform/eia861.py

```diff
@@ -837,10 +1064,10 @@ def transform(raw_dfs, eia861_tables=pc.pudl_tables["eia861"]):
                     "Not transforming EIA 861.")
         return tfr_dfs
     # for each of the tables, run the respective transform funtction
-    for table in eia861_transform_functions:
-        if table in eia861_tables:
-            logger.info(f"Transforming raw EIA 861 DataFrames for {table} "
-                        f"concatenated across all years.")
-            tfr_dfs[table] = _early_transform(raw_dfs[table])
-            eia861_transform_functions[table](tfr_dfs)
+    for table in eia861_tables:
+        logger.info(f"Transforming raw EIA 861 DataFrames for {table} "
+                    f"concatenated across all years.")
+        assert table in tfr_funcs.keys()
```

Comment:
Do you think it's worth creating a custom assertion error message here?

Reply:
Yeah definitely, I was just frustrated and wanted to get it checked in, and we're checking this differently across different data sources so I wasn't sure what the right thing to check was.

Reply:
Ok I'll add a comment so we remember to do so in the future.